utils/irqbalance/files/irqbalance.init

```diff
@@ -16,28 +16,36 @@ start_service() {
 	config_get_bool enabled irqbalance enabled 0
 	[ "$enabled" -gt 0 ] || return 0
 
-	# 2 is the default
 	config_get deepestcache irqbalance deepestcache 2
-
-	# 10 is the default
 	config_get interval irqbalance interval 10
-
-	# empty is the default
 	config_get banned_cpulist irqbalance banned_cpulist ''
+	config_get debug irqbalance debug 0
+
+	procd_open_instance "irqbalance"
+	if [ -n "$banned_cpulist" ]; then
+		procd_set_param env IRQBALANCE_BANNED_CPULIST="$banned_cpulist"
+	fi
+
+	procd_add_jail_mount_rw /proc
+	procd_set_param command /usr/sbin/irqbalance
+	procd_append_param command -f
+	procd_append_param command "-c $deepestcache"
```

Comment:
I’m so sorry I have overlooked this. Does it work as expected? I think it will result the same problem.
Generally speaking it should be `procd_append_param command -c "$deepestcache"`

Reply:
Passing "-t $interval" or "-i $1" work as expected, just tested to set the interval to 15, and also IRQ banning works (which was the original bug). So I assume that "-c $deepestcache" also works.

Reply:
Okay, thanks for clarification.